tests/src/core/testqgselevationmap.cpp

```diff
@@ -0,0 +1,107 @@
+/***************************************************************************
+  testqgselevationmap.cpp
+  --------------------------------------
+Date                 : August 2022
+Copyright            : (C) 2022 by Martin Dobias
+Email                : wonder dot sk at gmail dot com
+ ***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+#include "qgstest.h"
+
+#include "qgsapplication.h"
+#include "qgselevationmap.h"
+#include "qgsrasterlayer.h"
+#include "qgsrenderchecker.h"
+
+
+static QString _fileNameForTest( const QString &testName )
+{
+  return QDir::tempPath() + '/' + testName + ".png";
+}
+
+static bool _verifyImage( const QString &testName, QString &report )
```

Comment:
Can you use the base class method QgsTest::imageCheck here instead? I'm trying to consolidate all the duplicate render checking code into that base class